src/Components/Faq.jsx

```diff
@@ -1,92 +0,0 @@
-"use client";
-import React, { useState, useRef } from "react";
-import { IoIosArrowDown } from "react-icons/io";
-
-const Faq = ({ title, title2, content, index, isOpen, onClick }) => {
-    const contentRef = useRef();
-
-    return (
-        <li className="border-b border-zinc-400 ">
-            <h2
-                onClick={onClick}
-                className="flex  flex-row justify-between items-center font-medium text-zinc-700 p-4 cursor-pointer"
-            >
-                <span>{index + 1}. {title}</span>
-                <div className="flex items-center gap-2">
-                    <span>{title2}</span>
-                    <IoIosArrowDown className={`transform transition-transform duration-500 ${isOpen ? "rotate-180" : ""}`} />
-                </div>
-            </h2>
-            <div
-                ref={contentRef}
-                className={`overflow-hidden transition-all duration-500`}
-                style={{
-                    maxHeight: isOpen ? `${contentRef.current?.scrollHeight}px` : "0",
-                }}
-            >
-                <div className="text-zinc-600">
-                    <p className="p-3">{content}</p>
-                </div>
-            </div>
-        </li>
-    );
-};
-
-const Accordion = () => {
-    const [openTab, setOpenTab] = useState(null);
-
-    const toggleTab = (idx) => {
-        setOpenTab(openTab === idx ? null : idx);
-    };
-
-    const items = [
-        {
-            title: "What programs do you offer?",
-            content:
-                "We offer a variety of programs in fields such as IT, skilled trades, entrepreneurship, designed to prepare students for specific careers",
```

Comment:
This is static content please put it in a different file, maybe each item can have an id instead of using the index to keep track of the items